### PR TITLE
Created Cards and About Me

### PR DESCRIPTION
**Pull request by over-geek**
## Portfolio project milestone: Cards and About section

In this milestone, I:
- Added a cards section to markup.
- Added project screenshot to card
- Added some text to the card
- Created button class with hover properties.
- Added a button to the card section
- Styled the card section
- Added about section
- Added text and links to the about section
- Added a button to the about section
- Styled the about section
- Fixed linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
...
+                        </button>
+                    </div>
+                    <div class="skills">
+                        <p>Skills</p>
+                        <button>
+                            <img src="asset/chevron-right.png" alt="arrow icon">
+                        </button>
+                    </div>
+                </div>
+            </div>
```
Comment (levy002) [inline index.html:266]: - [ ] Kindly make sure to wrap the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`. :wink: 
- [ ] KIndly make sure to wrap the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.

File: index.html [hunk 2]
```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
...
+                    </div>
+                    <div>
+                        <p>
+                            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, don't hestiate to contact me.
+                        </p>
+                    </div>
+                    <div>
+                        <div>
+                            <p id="about-heading">Let's Connect</p>
```
Comment (levy002) [inline index.html:204]: - [ ] Kindly make sure to wrap the `LET'S CONNECT` title into the heading `hx` tag. :wink:

File: index.html [hunk 3]
```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
...
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot4.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Multi-Post Stories</p>
```
Comment (levy002) [inline index.html:160]: - [ ] Kindly make sure to wrap the title of the project into the heading `hx` tag. :wink:

File: index.html [hunk 4]
```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
...
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot3.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Tonic</p>
```
Comment (levy002) [inline index.html:128]: - [ ] Kindly make sure to wrap the title of the project into the heading `hx` tag. :wink:

File: index.html [hunk 5]
```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
...
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot2.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Multi-Post Stories</p>
```
Comment (levy002) [inline index.html:96]: - [ ] Kindly make sure to wrap the title of the project into the heading `hx` tag. :wink:

File: index.html [hunk 6]
```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
...
         </section>
+        <section id="card-section">
+            <div class="card-container">
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot1.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Tonic</p>
```
Comment (levy002) [inline index.html:64]: - [ ] Kindly make sure to wrap the title of the project into the heading `hx` tag. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @over-geek   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good UI :heavy_check_mark: 
- Professional PR :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @over-geek , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Nice UI ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**